src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,227 @@
+orientation (View Angle) and the Solid Angle, of each target pixel with 
+respect to the viewpoint. 
+<p> 
+A map containing a set of points 
+describing the locations of the observer and a digital elevation model 
+(DEM) are the mandatory inputs for <em>r.survey</em>. <i>It is very 
+importante that all observation points have different categories, i.e. 
+cat values (see the examples below)</i>. 
+In addition, maps of buildings 
+and trees, portraying height information, can be used to alter the DEM. 
+In the case of trees map, apart from the height value, a third column 
+in the attribute table has to contain the standard deviation value 
+related to the heigh. This is used to simulate the rougness of forested 
+areas. 
+<p>
+Many other options can be set according to the aim of the 
+analysis, such as: observer height respect to the ground or respect to 
+an elevation datum (absolute elevation), maximum distance to perform 
+the calculations, a view-angle threshold to exclude, a priori, cell 
+oriented almost perpendicularly to the lines of sight, the average size 
+of the observed object. The input points map must be provided to the 
+tool as a vector layer. In case it represents the positions of a UAV, 
+an helicopter or of a satellite, it must include an attribute field 
+containing absolute elevation values with respect to a vertical datum 
+(the same datum used by the input DEM). Observer height (respect to the 
+ground, the default is 1.75 m) and maximum distance of observation 
+(default is 1000 m)  are parameters needed by the r.viewshed module. 
+The object radius is used to approximate, with an equivalent circle, 
+the minimum size of an object centered in the cells center and oriented 
+according to the slope and aspect of the cells. Since r.survey produces 
+different types of outputs, a name to be used for the prefix of the 
+output maps is requested.  
+
+<p> 
+<p>
+
+3D Distance is the three 
+dimensional linear distance between a viewpoint and a target pixel (see 
+the illutratio bellow). The min3dDistance map portrays  the value of 
+the minimum three-dimensional distance between each pixel and the 
+closest viewpoint, in meters.
+<p>  
+
+<img src="r.survey_figures/fig_2.png" alt="e3Ddistance">  
+
+<p> 
+Given a 
+viewpoint, r.survey calculate the unit vector describing direction and 
+sense of the line of sight between that viewpoint and each visible 
+pixel (see illustration bellow). We define View Angle as the angle 
+between the unit vector representing the line of sight and the unit 
+vector normal to the terrain surface in each pixel. The maxViewAngle 
+map shows the value of the maximum View Angle between each pixel and 
+the viewpoints, in degrees. It is a measure of the most frontal view 
+each single cell is visible from.  View angle output is always larger 
+than 90&deg; and smaller than 180&deg;. 
+<p>  
+
+<img src="r.survey_figures/View_Angle.png" alt="view angle">   
```

Comment:
Please move the figure into the module directory (then the Makefile will install it, otherwise you need to modify the Makefile with a new rule).
The name should be then be `r.survey_view_angle.png` (so that it is clear to which module the PNG file belongs to).

Reply:
Please use a <div> ... box, see at the end of this section:

https://trac.osgeo.org/grass/wiki/Submitting/Docs#Images